src/core/client/systems/job.ts

```diff
@@ -95,6 +95,11 @@ class ObjectiveController {
             }
         }
 
+        if (isFlagEnabled(objective.criteria, JobEnums.ObjectiveCriteria.VEHICLE_ENGINE_OFF)) {
```

Comment:
I see this on client-side; but I also believe this should be checked on server-side.